ui/src/main/webapp/package.json

```diff
@@ -33,6 +33,7 @@
     "google-code-prettify": "1.0.1",
     "jquery": "2.1.4",
     "jquery-match-height": "0.7.0",
+    "ng2-file-upload": "^1.1.0",
```

Comment:
Well if they use semantic versioning correctly, which I suppose they do, this should be just for minor changes which should not have any BC changes. And new versions could have some bug fixes, so I usually use this. But it is true that Angular team didn't really respect it during those RC versions.